crates/matrix-sdk-common/src/deserialized_responses.rs

```diff
@@ -703,6 +706,50 @@ impl From<SyncTimelineEventDeserializationHelperV0> for SyncTimelineEvent {
     }
 }
 
+/// Represent a failed to send error of an event sent via the send_queue.
+/// These errors are not automatically retrievable, but yet some manual action
+/// can be taken before retry sending.
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Enum))]
+pub enum QueueWedgeError {
```

Comment:
Since this is related to the send queue, can we put it in the send queue file? Or is there a reason I'm missing to put it into deserialized_responses?

Also, should this use our common pattern of deriving `thiserror::Error`, to make it clear it's an error type, and would avoid the manual display impl? (uniffi also has an `uniffi::Error` derive which could be convenient here)

Reply:
Send Queue is in `matrix-sdk` and this error is persisted in the state storage `matrix-sdk-base`. 
So it cannot be in the send_queue.rs file.